dockerfile

```diff
@@ -0,0 +1,40 @@
+# Use an official Maven image as the build image
+FROM partnersinhealth/debian-build:latest AS build
+
+#build args
+ARG EMR_CONFIG=""
+ARG EMR_FRONTEND=""
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy project files to the container
+COPY ./pom.xml .
+COPY ./openmrs-distro.properties .
+COPY ./debian/ ./debian/
+COPY ./assembly.xml .
+COPY ./import_config.sh .
+
+# Install application using Maven
+RUN mvn -T 1C clean compile -Dmaven.test.skip -DskipTests -Dmaven.javadoc.skip=true
+RUN ./import_config.sh ${EMR_CONFIG} ${EMR_FRONTEND}
+
+# Use an official Tomcat image as the app image
+FROM tomcat:9-jdk8
+# Set the working directory in the container
+WORKDIR /app
+# Copy the built files from the build stage to the container
+COPY --from=build /app/target/distro/web/openmrs.war /usr/local/tomcat/webapps/openmrs.war
+COPY --from=build /app/target/distro/web/modules/ /app/.OpenMRS/modules/
+COPY --from=build /app/target/distro/web/owa/ /app/.OpenMRS/owa/
+COPY --from=build /app/frontend /app/.OpenMRS/frontend/
+COPY --from=build /app/configuration/ /app/.OpenMRS/configuration/
+COPY --from=build /app/configuration/frontend/ /app/.OpenMRS/frontend/site/
```

Comment:
It seems odd to use /app as a parent directory of _both_ the source build files and scripts _and_ the .OpenMRS directory.  I might be inclined to keep these more separate.

Reply:
Hi Mike,
These are folders are in different containers, one for build and one for runtime, as this is a multi-step build. Using /app for a container's working directory is just a common convention. For clarity, I think it may make since to switch the build containers working directory name.